packages/grid/_modules_/grid/components/containers/GridRootStyles.ts

```diff
@@ -229,7 +229,7 @@ export const useStyles = makeStyles(
           textOverflow: 'ellipsis',
           whiteSpace: 'nowrap',
           borderBottom: `1px solid ${borderColor}`,
-          '&:focus-within': {
+          '&:focus-within:not(.MuiDataGrid-cellCheckbox)': {
```

Comment:
Developers can move the focus inside their cell too: #1465. Could we use the editable CSS class name instead?

Reply:
I'm sorry it seems that I'm struggling to understand the end goal. I thought I'm only removing the outline for the cell that has a checkbox in it and keep it as it is for all the others, no?

Reply:
I'm sorry it seems that I'm struggling to understand the end goal. I thought I'm only removing the outline for the cell that has a checkbox in it and keep it as it is for all the others, no? If I use `.MuiDataGrid-cellEditable` it would remove all editable cells, also the checkbox cell doesn't currently have that class.

Reply:
My assumption are the following:

> a single focus-visible indicator when in view mode (**to** not be confused about where the real focus is), and always have the outline when in edit mode (this style works in a combination of the box-shadow, **to** make it clear what's in edit mode)

Reply:
> a single focus-visible indicator when in view mode (to not be confused about where the real focus is, only one element can have it, not two), and always have the outline when in edit mode (this style works in a combination of the box-shadow, to make it clear what's in edit mode)

@oliviertassinari  I'm missing something here. When you say focus-visible I think about the `:focus-visible` pseudo-class. It's only applied to one element and where the real focus is. I agree with showing the dotted outline in view mode and the solid outline in edit mode.